griptape/drivers/prompt/openai_chat_prompt_driver.py

```diff
@@ -79,7 +92,25 @@ def try_run(self, prompt_stack: PromptStack) -> TextArtifact:
         parsed_result = result.parse()
 
         if len(parsed_result.choices) == 1:
-            return TextArtifact(value=parsed_result.choices[0].message.content.strip())
+            message = parsed_result.choices[0].message
+            tool_calls: list[ChatCompletionMessageToolCall] = message.tool_calls
+
+            if tool_calls:
+                actions = [
+                    ActionsArtifact.Action(
+                        tag=tool_call.id,
```

Comment:
Can multiple `Action`s have the same `tag`? If so, should it be called `id` instead of `tag`? I might just generally be missing the reasoning behind each of the fields in `Action`